therapeutic/models/nh_clinical_patient_observation_therapeutic.py

```diff
@@ -0,0 +1,105 @@
+from openerp import models, api
+
+from openerp.addons.nh_observations import fields as obs_fields
+
+
+class NhClinicalPatientObservationTherapeutic(models.Model):
+    """
+    Model for the therapeutic observation.
+    """
+    _name = 'nh.clinical.patient.observation.therapeutic'
+    _inherit = 'nh.clinical.patient.observation'
+
+    _patient_status_selection = [
+        ('AW', 'Awake'),
+        ('CO', 'Consultation'),
+        ('ED', 'Education'),
+        ('GT', 'Group Therapy'),
+        ('LE', 'Leave'),
+        ('OU', 'Out (With Permission)'),
+        ('MI', 'Missing (Inform S/N)'),
+        ('SL', 'Sleeping'),
+        ('TR', 'Transfer'),
+        ('DI', 'Discharge')
+    ]
+
+    # This field is left out of the form description because its type is not
+    # currently supported. This is a good thing as we do not want to display
+    # this field at the moment. It is maintained for use of future features.
+    patient_level = obs_fields.Many2one(
```

Comment:
I wouldn't add this at this point in time. While we are not using it now, it means this won't be reviewed when the implementation of that features comes into place.

Reply:
I added this because we agreed to do so in 3 amigos. Its probably my fault though because I had a note to add the scenario to the story and forgot to do so. I'll remove it anyway though.